hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/compact/SparkScheduleCompactionActionExecutor.java

```diff
@@ -90,4 +99,13 @@ protected HoodieCompactionPlan scheduleCompaction() {
     }
   }
 
+  public Long parseTs(String time) {
+    Long timestamp = null;
+    try {
+      timestamp = HoodieActiveTimeline.COMMIT_FORMATTER.parse(time).getTime() / 1000;
+    } catch (ParseException e) {
+      e.printStackTrace();
```

Comment:
we'd better throw an 'HoodieCompactionException' here

Reply:
ok